src/taco-cli/cli/kit.ts

```diff
@@ -721,7 +721,7 @@ class Kit extends commands.TacoCommandBase {
         });
     }
 
-    private static select(commandData: commands.ICommandData): Q.Promise<tacoUtility.ICommandTelemetryProperties> {
```

Comment:
because we should avoid full static implementations. they can't be extended. for e.g. for remote say I want to add "taco macincloud add". I would want to extend existing implementation.